### PR TITLE
BAT-7798 - Initial scripts

### PR DESCRIPTION
**Pull request by asiebert**
:arrow_upper_left: https://work4labs.atlassian.net/browse/BAT-7798 

-----

cf. CircleCi builds for tests

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        background: true
+
+  cache_directories:
+    - "~/cache"
+
+
+test:
+  override:
+    # CCi may have trouble with some commands, hence the dummy var 
+    - DUMMY="var" && [[ $(mongo --quiet --eval "db.version()") == "3.4.9" ]]
```
Comment (olivierlefloch) [inline circle.yml]: Use MONGO_VERSION here?

File: bin/setup_elasticsearch_1_3_4.sh [hunk 2]
```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+set -e
+
+############################
+# ELASTICSEARCH 1.3.4 SETUP
+############################
+# See README.md for usage
+
+ES_VERSION=1.3.4  # Non-generic script - source URL is different depending on the version
```
Comment (olivierlefloch) [inline bin/setup_elasticsearch_1_3_4.sh]: Still, this makes it needlessly hard to maintain, track history, etc.

File: circle.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    - sleep 5 && [[ $(curl -s http://localhost:9200/ | grep number) == *1.3.4* ]]
```
Comment (olivierlefloch) [inline circle.yml]: I still think you should have an ENV VAR for the ES version, even if the url changes with each version, etc. Will still mean less code to update as things change.

File: bin/setup_elasticsearch_1_3_4.sh (hunk 2 above)
Comment (asiebert) [reply bin/setup_elasticsearch_1_3_4.sh]: As-is, my alternative is to crash if requesting a version >2... in fact, I don't know at which version does the URL changes, hence I went for freezing it altogether.

I'm not too worried about history: if at any point we modify (and rename) this script to support arbitrary versions, the history should be preserved – as long as we update this one, and not copy/paste it for a different version.

File: circle.yml (hunk 1 above)
Comment (asiebert) [reply circle.yml]: I'll check if I can, not entirely sure.

**Comment by olivierlefloch** [comment]
I think it’s just really confusing. And it’s fine if the script crashes
with a version >2..?

File: bin/setup_elasticsearch_1_3_4.sh (hunk 2 above)
Comment (asiebert) [reply bin/setup_elasticsearch_1_3_4.sh]: > I think it’s just really confusing. And it’s fine if the script crashes with a version >2..?

File: circle.yml (hunk 1 above)
Comment (asiebert) [reply circle.yml]: Seems to be working! :) 81d4e7c  :white_check_mark:

File: bin/setup_elasticsearch_1_3_4.sh (hunk 2 above)
Comment (asiebert) [reply bin/setup_elasticsearch_1_3_4.sh]: Looking at it again, I guess that's fair. 3ab38d4 :white_check_mark:

File: circle.yml (hunk 3 above)
Comment (asiebert) [reply circle.yml]: 3ab38d4 :white_check_mark:

**Pull request merged**